cloudbuild.yaml

```diff
@@ -7,14 +7,18 @@ timeout: 1200s
 options:
   substitution_option: ALLOW_LOOSE
 steps:
+  # The image must contain bash and curl. Ideally it should also contain
+  # the desired version of Go (currently defined in release-tools/travis.yml),
+  # but that just speeds up the build and is not required.
   - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20200421-a2bf5f8'
-    entrypoint: make
+    entrypoint: bash
     env:
-    - PULL_BASE_REF=$_PULL_BASE_REF
+    - GIT_TAG=${_GIT_TAG}
+    - PULL_BASE_REF=${_PULL_BASE_REF}
+    - STAGING_PROJECT=${_STAGING_PROJECT}
```

Comment:
You are right, setting `REGISTRY_NAME` directly is cleaner. I'm not even sure why we have `_STAGING_PROJECT` under substitutions.

@vitt-bagal: do you remember where that came from? Isn't that always `k8s-staging-csi`?

Reply:
> @vitt-bagal: do you remember where that came from? Isn't that always k8s-staging-csi?

It came from this [commit](https://github.com/kubernetes-csi/node-driver-registrar/commit/a44583857c746a1de0cc26a74e2ab1446adf3b65). I think this can be removed safely if we set `REGISTRY_NAME` directly

Reply:
If we can move `cloudbuild.yaml` into `release-tools` and just symlink to it from the root directory of a project (similar to `.travis.yml -> release-tools/travis.yml`), then it may make sense to have the substitution because different projects might have different staging projects.